substrait_consumer/tests/functional/extension_functions/arithmetic_snapshots/DataFusionProducer/atan2_plan.json

```diff
@@ -3,14 +3,14 @@
     {
       "extensionFunction": {
         "extensionUriReference": 4294967295,
-        "name": "atan2"
+        "functionAnchor": 1,
```

Comment:
The output order of the extension functions flipped but the values are the same.  Normalization of the URI and function sections is probably useful to avoid diffs at runtime.  Here we probably want to normalize so that we don't have to resubmit the results if the order changes in a subsequent run.